### PR TITLE
Mlt 205 neg ratio pyveda

### PR DESCRIPTION
**Pull request by glortho**
What does this PR do?
---------------------

- Adds support for specifying background_ratio to pyveda when creating data set.
- Includes test for create_from_geojson.

What steps, if any, do reviewers need to take to set up their environments to test this properly?
---------------------

- n/a

Acceptance criteria
---------------------

- When you create a data set, the doc should be created either with the `background_ratio` you specified or the default of 0.0.

Known issues
---------------------

- The corresponding warmer work isn't done yet.

:white_check_mark: Checklist
---------------------

- [x] No additional work is in process. This PR should be ready for merge right now.
- [x] Tests are included
- [x] There is at least one assignee

### CODE REVIEW

**Comment by rjpolackwich** [comment]
so you HAVE to spec a background ratio when you create a dataset?

**Comment by glortho** [comment]
@rjpolackwich it should default to 0 if you don't. but yeah otherwise i don't think there's support for adding it later. should we add that?

**Comment by rjpolackwich** [comment]
@glortho gothcha. Reason I ask is bc to me it doesn't necessarily seem natural to define something like that when first "creating" a dataset. That param becomes relevant dep on the problem type being solved (which involves model type etc)

**Comment by chelm** [comment]
This is looking correct. Background ratio is caching parameter, not a client side param that we will most likely be looking into adding. 

@glortho @chloe-hampton Can you, or did you, add support for background_ratio to `append_from_geojson`? It'll be needed there as well.

File: tests/unit/test_main.py [hunk 1]
```diff
@@ -70,5 +73,44 @@ def test_loadstore(self):
...
+            def __init__(self):
+                self.dtype = np.dtype('uint8')
+                self.shape = [256, 256, 3]
+                self.rda = _Rda()
+                self.rda_id = 'someotherid'
+
+        image = _Image()
+
+        vcp = pv.create_from_geojson(geojson, image, mock_doc['name'])
+        self.assertTrue(isinstance(vcp, VedaCollectionProxy))
```
Comment (rjpolackwich) [inline tests/unit/test_main.py]: python unit has all the checks builtin: use `assertIsInstance(...)` for these kind of checks; failures will report more accurately

**Comment by rjpolackwich** [comment]
Nice job mocking functions in the test

File: tests/unit/test_main.py (hunk 1 above)
Comment (chloe-hampton) [reply tests/unit/test_main.py]: cool I can correct that

**Comment by chloe-hampton** [comment]
@chelm it's not currently in `append_from_geojson`. That's a good catch.

**Pull request merged**